addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -2351,15 +2351,17 @@ def test_payment_move_state(self):
             ('out_receipt', 1000.0, [('out_refund', 1000.0)], 'reversed'),
             ('out_receipt', 1000.0, [('out_refund', 500.0), ('out_refund', 500.0)], 'reversed'),
             ('out_receipt', 1000.0, [('reverse', 1000.0)], 'reversed'),
-            ('out_refund', 1000.0, [('reverse', -1000.0)], 'reversed'),
+            ('out_refund', 1000.0, [('reverse', -1000.0)], 'paid'),
+            ('out_refund', 1000.0, [('entry', 1000.0)], 'reversed'),
```

Comment:
I don't understand the changes here. I was expecting to see a `out_invoice`/`in_invoice` somewhere here :grimacing: 

Can you explain please ?

Reply:
When reversing a `in_refund/out_refund` with `in_invoice/out_invoice` and reconciling both we end up with `refund payment state == 'paid'`, and when reversing with `entry`, `refund payment state == 'reversed'`